source/manual/alerts/high-priority-tests.html.md

```diff
@@ -1,15 +1,19 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Run high priority tests
+title: Smokey loop tests
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
 last_reviewed_on: 2020-06-15
 review_in: 6 months
 ---
 
-The high priority tests [come from Smokey][smokey] and [the Icinga check is defined in Puppet][icinga].
-Smokey can fail with an error message "Run high priority tests", or simply with "Smokey failed".
+[Smokey][smokey] runs in a continuous loop in each environment.
+We have [Icinga checks][] for [most Smokey features][], so that
```

Comment:
We don't need the extra `[]` characters here - it defaults to using the link text as the link alias value.

Reply:
Ahhh, I was sticking to the known-to-work approach. Updated to drop them 👍